src/marketplace/sellers/account-setup.md

```diff
@@ -6,6 +6,39 @@ redirect_from: /marketplace/sellers/account-setup-overview.html
 
 Create a new Marketplace account, or use your existing Magento account. Then, use the Marketplace Developer Portal to manage your profile information, product submissions, and the monthly payments you receive from sales on Magento Marketplace.
 
+## Bulleted list {#bulleted-list}
+
 -  [Process overview]({{ site.baseurl }}/marketplace/sellers/account-setup-process.html)
 -  [Register as a developer]({{ site.baseurl }}/marketplace/sellers/developer-register.html)
 -  [Marketplace developer portal]({{ site.baseurl }}/marketplace/sellers/developer-portal.html)
+
+## Relative links {#relative-links}
+
+-  [Process overview]({% link marketplace/sellers/account-setup-process.md %})
+-  [Register as a developer]({% link marketplace/sellers/developer-register.md %})
+-  [Marketplace developer portal]({% link marketplace/sellers/developer-portal.md %})
+
+## Absolute links {#absolute-links}
+
+-  [Process overview](https://devdocs.magento.com/marketplace/sellers/account-setup-process.html)
+-  [Register as a developer](https://devdocs.magento.com/marketplace/sellers/developer-register.html)
+-  [Marketplace developer portal](https://devdocs.magento.com/marketplace/sellers/developer-portal.html)
+
+## Reference Links {#reference-links}
+
+
+Lorem ipsum dolor sit amet, [process overview] sed cibo graeco inermis ex. Doctus labitur assueverit ut quo, ei vero adipisci sit. Accusata partiendo eloquentiam no nam, ad duo graeci splendide. Mei dicta everti feugait eu, [register] putent vidisse dolorum has ea. Mea odio ipsum malorum cu, sit cu cetero interesset, et usu sumo cetero intellegam. Mei ceteros splendide appellantur id, [portal] et intellegat temporibus eam.
```

Comment:
Can this be done in English?

Reply:
That's Lorem Ipsum - dummy text - not sure why it's been added @ccondra?

Reply:
Blank lines are causing tests to fail